example/extensions/lib_custom_op/transposerowsp_lib.cc

```diff
@@ -115,6 +118,8 @@ MXReturnValue inferType(std::map<std::string, std::string> attrs,
 MXReturnValue inferSType(std::map<std::string, std::string> attrs,
                         std::vector<int> &instypes,
                         std::vector<int> &outstypes) {
+  if (instypes[0] != kRowSparseStorage) 
+    return MX_FAIL;
```

Comment:
error message telling users they provided the wrong storage type?

Reply:
Added.